config/build_archer2.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+BUILD_MODULES=config/load_archer2.sh
+
+#module purge
+#echo "Purging current module set"
+echo "Sourcing file: $BUILD_MODULES to build QMCPACK"
+
+. $BUILD_MODULES
+
+module list
+
+echo "Either source $BUILD_MODULES or load these same modules to run QMCPACK"
+
+#export BLAS_LIBS="-L$OLCF_OPENBLAS_ROOT/lib -lopenblas"
+#export LAPACK_LIBS="$BLAS_LIBS $OLCF_NETLIB_LAPACK_ROOT/lib64/liblapack.a"
+
+declare -A builds=( ["cpu"]="-DBUILD_PPCONVERT=1" \
+                    ["complex_cpu"]="-DQMC_COMPLEX=1" \
+#		    ["legacy_gpu"]="-DQMC_CUDA=1 " \
+#		    ["complex_legacy_gpu"]="-DQMC_CUDA=1 -DQMC_COMPLEX=1 " \
+		  )
+
+mkdir bin
+
+for build in "${!builds[@]}"
+do
+    echo "building: $build with ${builds[$build]}"
+    rm bin/qmcpack_${build}
+    mkdir build_${build}
+    cd build_${build}
+    cmake -DCMAKE_C_COMPILER="cc" \
+          -DCMAKE_CXX_COMPILER="CC" \
+	  -DCMAKE_SYSTEM_NAME=CrayLinuxEnvironment \
+          -D LIBXML2_LIBRARY="$LIBXML2_ROOT/lib/libxml2.so" \
```

Comment:
Does `-DLibXml2_ROOT=$LIBXML2_ROOT` work?

Reply:
Dear @ye-luo,
yes, `-DLibXml2_ROOT=$LIBXML2_ROOT` works.

Reply:
Dear @ye-luo and @prckent ,
I made the modifications above, and pushed them on https://github.com/zenandrea/qmcpack
What should I do now? A new pool request?

Reply:
Andrea - This is probably clear after the changes you pushed, but if you push changes to a branch used in a PR (say, your `develop` in this case), then the PR automatically updates with the commit visible and whatever changes and new/removed files were in the commit. You don't need to make a new pull request. Generally people make a named branch for whatever they are working on, say `archer2` in this case, and then delete this development branch after they changes get successfully merged into QMCPACK/develop .